README.md

```diff
@@ -1,2 +1,186 @@
+
+```
+ruby person.rb
+ruby student.rb
+ruby teacher.rb
+```
+
+Run the execution file
+
+```
+./main.rb
+```
+
+3. The output will be printed on the console.
+
+### Run tests
+
+To run the test change the file name for respective file
+
+```
+ ruby -S rspec spec/<filename>_spec.rb
+
+```
+
+### Deployment
+
+There is no specific deployment process for this project as it is a code implementation. However, you can integrate project into your own Ruby applications.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="authors"></a>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Atif Zada**
+- GitHub: [@Atif-Zada](https://github.com/AtifZada)
+- LinkedIn: [@Atif-Zada](https://linkedin.com/in/AtifZada)
+
+
+👤 **Spencer Okyere**
+
+- GitHub: [@skabeo](https://github.com/skabeo)
+- Twitter: [@black_okyere](https://twitter.com/black_okyere)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/spencer-okyere)## 🔭 Future Features <a name="future-features"></a>
+
+Possible future features for this project include:
+
+- Add more methods to the `TDD Project`
+- Enhance error handling and edge case scenarios
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Atifzada/Test_Driven_Development-TDD-/issues).
+
+If you would like to contribute to this project, you can follow these steps:
+
+1. Fork the repository.
+2. Create a new branch for your changes.
+3. Make your changes and commit them.
+4. Push your changes to your forked repository.
+5. Create a pull request back to the original repository.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please give it a star on GitHub.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- [Ruby](https://www.ruby-lang.org/en/)
+- [GitHub](https://github.com/)
+- [Microverse](https://www.microverse.org/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **Can I modify and redistribute this project?**
+
+  - Yes, you can modify and redistribute this project as long as you follow the terms of the MIT license.
+
+- **How can I contribute to this project?**
+
+  - Contributions, issues, and feature requests are welcome! You can check the issues page to see if there are any current issues or feature requests that you can work on. If not, feel free to submit a new issue or pull request. Before contributing, please read the CONTRIBUTING.md file for guidelines on how to contribute to this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] the link to the License is not working because the file is called LICENSE.md and you're referring to LICENSE please make sure to fix this.

Reply:
sure I will do it